modules/p2/ding_interaction/plugins/content_types/interaction_pane.inc

```diff
@@ -54,7 +48,7 @@ function ding_interaction_content_type_render($subtype, $conf, $panel_args, $con
     }
   }
 
-  $data = ding_interaction_content($module_list, $conf['count'], $extra);
+  $data = ding_interaction_content($module_list, variable_get('ding_interaction_page_count', DING_INTERACTION_DEFAULT_COUNT), $extra);
```

Comment:
If we do not want to support per pane count then we should remove the option to set it from the pane configuration and the pane title - as far as I can tell this is the only other place where it is used.